spec/truffle/signal_exception_spec.rb

```diff
@@ -44,4 +35,13 @@
     $?.termsig.should == Signal.list["KILL"]
     output.should == "hello\n"
   end
+
+  it "cannot be trapped with Signal.trap" do
+    ruby_exe(<<-RUBY)
+      Signal.trap("PROF") {}
+      raise(SignalException, "PROF")
+    RUBY
+
+    $?.termsig.should == Signal.list["PROF"]
```

Comment:
`Signal.list.fetch("PROF")` would be safer here.
Same for other specs.

Reply:
Fixed.